target/linux/ar71xx/files/arch/mips/ath79/Kconfig.openwrt

```diff
@@ -837,6 +837,18 @@ config ATH79_MACH_GS_MINIBOX_V1
 	select ATH79_DEV_USB
 	select ATH79_DEV_WMAC
 
+config ATH79_MACH_GS_MINIBOX_V32
```

Comment:
this whole block is not properly indented

Reply:
right